network/dag/bbolt_dag.go

```diff
@@ -94,7 +94,11 @@ func (d dataSizeStatistic) String() string {
 
 // NewBBoltDAG creates a etcd/bbolt backed DAG using the given database.
 func NewBBoltDAG(db *bbolt.DB) DAG {
-	return &bboltDAG{db: db}
+	result := &bboltDAG{db: db}
```

Comment:
this is an additional feature in a rename branch?

Reply:
Will rename the bucket in a separate PR